packages/fuels/tests/logs.rs

```diff
@@ -1441,3 +1441,37 @@ async fn can_configure_decoder_for_script_log_decoding() -> Result<()> {
 
     Ok(())
 }
+
+// String slices can not be decoded from logs as they are encoded as ptr, len
+// TODO: Once https://github.com/FuelLabs/sway/issues/5110 is resolved we can remove this
+#[tokio::test]
+async fn test_string_slice_log() -> Result<()> {
+    abigen!(Contract(
+        name = "MyContract",
+        abi = "packages/fuels/tests/logs/contract_logs/out/debug/contract_logs-abi.json"
+    ));
+
+    let wallet = launch_provider_and_get_wallet().await;
+
+    let contract_id = Contract::load_from(
+        "tests/logs/contract_logs/out/debug/contract_logs.bin",
+        LoadConfiguration::default(),
+    )?
+    .deploy(&wallet, TxParameters::default())
+    .await?;
+
+    let contract_instance = MyContract::new(contract_id.clone(), wallet.clone());
+
```

Comment:
can't all this process be replaced by the `setup_test_program` macro?

Reply:
Nice! Yes it can. I just copied a random test. I will fix the other test to.